sources/packages/web/src/views/aest/ApplicationDetails.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Student Details</a
+    >
+  </h5>
+  <full-page-container v-if="applicationDetail.data">
```

Comment:
I thought ` <full-page-container` is a root container

Reply:
I inherited this pattern from application details screen where arrows are outside the v-container. Anyhow we do not have the wire-frame for these screens yet. we can follow the wireframes once they are out.

Reply:
I did not create any common component for header portion.